testing/test_cacheprovider.py

```diff
@@ -14,6 +14,12 @@
 pytest_plugins = ("pytester",)
 
 
+@pytest.fixture(scope="function", autouse=True)
+def handle_env(monkeypatch):
+    """Ensure env is like most of the tests expect it, i.e. not using tox."""
+    monkeypatch.delenv("TOX_ENV_DIR", raising=False)
```

Comment:
This only gets used in this module/file then, right?
Might be worth requesting it explicitly though, but there are like 12+ files failing due to this.